services/frontend-service/package.json

```
@@ -81,5 +81,5 @@
       "last 1 safari version"
     ]
   },
-  "proxy": "http://kuberpult-frontend-service:8081"
+  "proxy": "http://localhost:8081"
```

@leandro-freiheit please return this to 
```suggestion
  "proxy": "http://kuberpult-frontend-service:8081"
```